MAD.Procore.StudioHoursUpload/MAD.Procore.StudioHoursUpload.csproj

```diff
@@ -8,8 +8,8 @@
 
   <ItemGroup>
     <PackageReference Include="Dapper" Version="2.0.90" />
-    <PackageReference Include="MAD.API.Procore" Version="1.0.3" />
-    <PackageReference Include="MAD.Integration.Common" Version="1.1.14.3" />
+    <PackageReference Include="MAD.API.Procore" Version="2.0.1.2111030544" />
```

Comment:
Please revert this back to v1.0.3, don't want to accidently introduce any breaking changes with v2 of the Procore lib